src/compiler/scala/tools/nsc/ast/parser/Scanners.scala

```diff
@@ -103,11 +124,11 @@ trait Scanners extends ScannersCommon {
     }
   }
 
-  abstract class Scanner extends CharArrayReader with TokenData with ScannerData with ScannerCommon {
+  abstract class Scanner extends CharArrayReader with TokenData with ScannerData with ScannerCommon with DocScanner {
     private def isDigit(c: Char) = java.lang.Character isDigit c
 
     private var openComments = 0
-    protected def putCommentChar(): Unit = nextChar()
+    final protected def putCommentChar(): Unit = { processCommentChar(); nextChar() }
```

Comment:
Making this method final is a source incompatible change, which are supposed not to be done in minor Scala versions afaik. Concretely it broke the IDE. We can't update from 2.12.0 to 2.12.1. Since 2.12.1 is already released it is too late to revert the change. We may find a workaround for it but I wanted to mention in anyway.